version/constants.go

```diff
@@ -109,6 +109,12 @@ var (
 		constants.MainnetID: time.Date(10000, time.December, 1, 0, 0, 0, 0, time.UTC),
 		constants.FujiID:    time.Date(10000, time.December, 1, 0, 0, 0, 0, time.UTC),
 	}
+
+	EForkTimes = map[uint32]time.Time{
+		constants.MainnetID: time.Date(10000, time.December, 1, 0, 0, 0, 0, time.UTC),
+		constants.FujiID:    time.Date(10000, time.December, 1, 0, 0, 0, 0, time.UTC),
+	}
+	TempForkTime = time.Date(10000, time.December, 1, 0, 0, 0, 0, time.UTC)
```

Comment:
not yet active, even in test setup. Will drop this as soon as I sufficiently code this stuff